src/public/index.html

```diff
@@ -10,6 +10,9 @@
 
 <body>
   <h1>The server works!</h1>
+  <h2>
+    <a href="https://en.wikipedia.org/wiki/Yeehaw_Junction,_Florida">Yeehaw! . . . Junction</a>
+  </h2>
 </body>
 
-</html>
+</html>
```

Comment:
This is an editor difference—standard editors will always end the file on a newline. For some reason, yours didn’t.

I’d recommend setting VS Code to always end on a newline, just because it’s a little more standard. In my experience, some things have actually broken because of it, but it’s pretty rare/freakish things. But just wanted to call attention to it: sometimes it matters; usually not, but something to be aware of.

Reply:
ahh, good to know.  That would have taken a while to debug the first time I came across it.  

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline